src/rev_ai/custom_vocabularies_client.py

```diff
@@ -60,11 +60,20 @@ def get_custom_vocabularies_information(self, id):
         :param id: string id of custom vocabulary submission
         """
 
-        response = self._make_http_request(
-            "GET",
-            urljoin(self.base_url, id)
-        )
+        response = self._make_http_request("GET", urljoin(self.base_url, id))
+        return response.json()
+
+    def get_list_of_custom_vocabularies(self, limit=None):
+        """ Get a list of custom vocabularies
+        :param limit: optional, limits the number of jobs returned,
+                      if none, a default of 100 jobs is returned, max limit if 1000
```

Comment:
Please remove the literal value from this line, we don't want it to get out of sync with the REST API docs.

Reply:
I think it would be more beneficial to link to the documentation rev.ai/docs/streaming